playground/backend/internal/cache/redis/redis_cache_test.go

```diff
@@ -535,6 +555,28 @@ func Test_unmarshalBySubKey(t *testing.T) {
 			want:    output,
 			wantErr: false,
 		},
+		{
+			// Test case with calling unmarshalBySubKey method with Canceled subKey.
+			// As a result, want to receive false.
```

Comment:
Why it will be false?

Reply:
@daria-malkova because `canceledValue` is JSON encoding of the false.